api/schemas/artist.py

```diff
@@ -33,6 +34,7 @@ class ArtistSchema(ArtistBase):
 class ArtistFull(ArtistSchema):
     media: list[MediaSchema]
     payment_urls: list[PaymentSchema]
+    social_links: list[SocialSchema]
```

Comment:
Shouldn't these all technically be `Optional[{SchemaName}]`, since they don't *need* to be populated?
```suggestion
    media: list[Optional[MediaSchema]]
    payment_urls: list[Optional[PaymentSchema]]
    social_links: list[Optional[SocialSchema]]
```
(This all assumes that I actually understand how Optional works, but I think this would allow for each of the fields to either have an empty list state, which would be its default value, or a list with one or more elements of the listed schema type.)